src/controllers/userController.ts

```diff
@@ -76,33 +69,29 @@ export class UserController {
       }
     } catch (err) {
       console.log(err);
-      return res.status(StatusCodes.INTERNAL_SERVER_ERROR);
+      return res.status(StatusCodes.INTERNAL_SERVER_ERROR).send(ReasonPhrases.INTERNAL_SERVER_ERROR);
     }
   }
 
   //Provides logged in user data to the client
   public getProfile(req: Express.Request, res: Express.Response) {
     const { user } = <getProfileDto>matchedData(req);
     if (user) {
-      res.send(user);
+      return res.send(user);
     } else {
-      res.status(StatusCodes.UNAUTHORIZED).send(ReasonPhrases.UNAUTHORIZED);
+      return res.status(StatusCodes.UNAUTHORIZED).send(ReasonPhrases.UNAUTHORIZED);
     }
   }
 
   //Provides a list of all users

```

Comment:
Remove comments - we don't write comments if there is no complex logic in code. We use descriptive names for methods i.e. `getAllUsers` - thanks to that we know what the functionality of the method is.